ci/dev/watch.ts

```diff
@@ -86,28 +84,21 @@ class Watcher {
         cleanup(code)
       })
     }
-    const bundle = bundler.watch((err: FixMeLater, buildEvent: FixMeLater) => {
-      if (err) {
-        console.error(err)
-        Watcher.log("parcel watcher terminated unexpectedly")
-        cleanup(1)
-      }
-
-      if (buildEvent.type === "buildEnd") {
-        console.log("[parcel] bundled")
-      }
-
-      if (buildEvent.type === "buildError") {
-        console.error("[parcel]", err)
-      }
-    })
 
     vscode.stderr.on("data", (d) => process.stderr.write(d))
     tsc.stderr.on("data", (d) => process.stderr.write(d))
     if (plugin) {
       plugin.stderr.on("data", (d) => process.stderr.write(d))
     }
 
+    const browserFiles = [
+      path.join(this.rootPath, "out/browser/register.js"),
+      path.join(this.rootPath, "out/browser/pages/login.js"),
+      path.join(this.rootPath, "out/browser/pages/vscode.js"),
+    ]
+
+    bundleBrowserCode(browserFiles)
```

Comment:
I think we could remove this since at this point `tsc` most likely won't have finished compiling yet.